### ISSUE TITLE
Rule UUF_UNUSED_FIELD does not report the line number correctly

### ISSUE THREAD

**Issue by kloessst** [spotbugs/spotbugs#1351]
Hello,
it seems like the rule `UUF_UNUSED_FIELD` does not report the start line under the `<SourceLine>` element. Here a minimal Java example:

```
package com.stefan.staticCodeAnalysis;

public class App {
    private int unused;
}
```

The following <BugInstance>  report is generated:

```
<BugInstance instanceOccurrenceNum='0' instanceHash='4264f97ab2ab7cd3d3984399d04ae6c8' rank='18' abbrev='UuF'
                 category='PERFORMANCE' priority='2' type='UUF_UNUSED_FIELD' instanceOccurrenceMax='0'>
        <ShortMessage>Unused field</ShortMessage>
        <LongMessage>Unused field: com.stefan.staticCodeAnalysis.App.unused</LongMessage>
        <Class classname='com.stefan.staticCodeAnalysis.App' primary='true'>
            <SourceLine classname='com.stefan.staticCodeAnalysis.App' start='3' end='3'
                        sourcepath='com/stefan/staticCodeAnalysis/App.java' sourcefile='App.java'>
                <Message>At App.java:[line 3]</Message>
            </SourceLine>
            <Message>In class com.stefan.staticCodeAnalysis.App</Message>
        </Class>
        <Field isStatic='false' classname='com.stefan.staticCodeAnalysis.App' signature='I' name='unused'
               primary='true'>
            <SourceLine classname='com.stefan.staticCodeAnalysis.App'
                        sourcepath='com/stefan/staticCodeAnalysis/App.java' sourcefile='App.java'>
                <Message>In App.java</Message>
            </SourceLine>
            <Message>Field com.stefan.staticCodeAnalysis.App.unused</Message>
        </Field>
        <SourceLine synthetic='true' classname='com.stefan.staticCodeAnalysis.App'
                    sourcepath='com/stefan/staticCodeAnalysis/App.java' sourcefile='App.java'>
            <Message>In App.java</Message>
        </SourceLine>
</BugInstance>
```
The direct child `SourceLine` element does not have a `start` attribute. The xdoc XML report generated by the Maven plugin is also not able to report the line correctly because of the missing attribute.
